src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -256,10 +256,13 @@ javac.opt.Xlint.desc.unchecked=\
     Warn about unchecked operations.
 
 javac.opt.Xlint.desc.varargs=\
-    Warn about potentially unsafe vararg methods
+    Warn about potentially unsafe vararg methods.
 
 javac.opt.Xlint.desc.preview=\
-    Warn about use of preview language features
+    Warn about use of preview language features.
+
+javac.opt.Xlint.desc.synchronization=\
+    Warn about synchronization attempts on instances of @ValueBased classes.
```

Comment:
The annotation java.internal.ValueBased is purely internal and should not show up in error messages.
The reference to value based should be to the ValueBased.html or just be a lower case value-based.

Reply:
Good point. Fixed two places to say value-based consistently since this is the language mentioned by ValueBased.html.